lib/ord.js

```diff
@@ -43,7 +42,6 @@ const fInitializeGlobal = (csn) => {
     const fServicesFilter = (keyDefinition) => keyDefinition.kind === "service" && !keyDefinition['@cds.external'];
     const fODMEntityFilter = (key, keyDefinition) => {
         return keyDefinition.kind === "entity"
-            && key.includes(capNamespace)
```

Comment:
This wasn't correct in case when the `namespace` is not defined in the service definition file. In that case, the `capNamespace` will be **undefined** and this filter will be wrong.
Another option would be: `&& (capNamespace && key.includes(capNamespace)` instead of this line.
@Fannon @zongqichen @RamIndia : Which option do you prefer(if any)?

Reply:
Hi @aramovic79 , do you know why there are two namespaces and what is the context behind them?

Reply:
> Hi @aramovic79 , do you know why there are two namespaces and what is the context behind them?

AFAIK, the `capNamespace` is a namespaces as given in the service.cds definition file. If it is not given(which can be the case), this value is undefined, of course; on the other side, `namespace` is the value given in the .cdsrc.json file and as a fallback, we are using name of the package from the package.json file. 
cc: @RoshniNaveenaS @RamIndia @Fannon

Reply:
Thanks, I'm confused when namespace are different in definition file, .cdsrc.json and package.json. I also need other opinion.